.github/workflows/python-app.yml

```diff
@@ -46,7 +46,9 @@ jobs:
 
     - name: Test with pytest
       run: |
-        DISPATCHER_MOCK_KB=yes python -m coverage run --source=cdci_osa_plugin -m pytest tests -sv --full-trace --log-cli-level=DEBUG 
+        DISPATCHER_MOCK_KB=yes  python -m coverage run --source=cdci_osa_plugin -m pytest tests -sv --full-trace --log-cli-level=DEBUG -m "not plugin_test_conf"
```

Comment:
Is there any problem if both tests are combined?

Separating them like that breaks coverage (that's why it decreased). It is possible to combine the coverage, [e.g.](https://stackoverflow.com/questions/26214055/combine-python-coverage-files) but why not keep it simple in one job?

Reply:
Is there a way to apply an env var only for certain tests? In our case would only for those tests that are marked with `plugin_test_conf`

Reply:
hm, I did not use it before, but [this](https://aalvarez.me/posts/pytest-tricks-for-better-python-tests/) seems to work

```python
import pytest


@pytest.fixture(autouse=True)
def env_setup(monkeypatch):
    monkeypatch.setenv('MY_SETTING', 'some-value')
    monkeypatch.setenv('ANOTHER_SETTING', 'some-value')
```